http/instances.go

```diff
@@ -29,23 +39,22 @@ func handleInstancesGet(ctx context.Context, core *cloudbrain.Core, w http.Respo
 	// Determine the path...
 	prefix := "/instances/"
 	if !strings.HasPrefix(r.URL.Path, prefix) {
-		respondError(ctx, w, http.StatusNotFound, nil)
+		respondError(ctx, w, http.StatusNotFound, errNoURLPrefix)
 		return
 	}
 	path := r.URL.Path[len(prefix):]
 	if path == "" {
-		respondError(ctx, w, http.StatusNotFound, nil)
+		respondError(ctx, w, http.StatusNotFound, errNoURLPath)
 		return
 	}
 
 	instance, err := core.GetInstance(ctx, path)
 	if err != nil {
-		cbcontext.LoggerFromContext(ctx).WithField("err", err).Error("couldn't get instance")
```

Comment:
Should we maybe keep this log message? Usually I think an error should only be handled once ("log it or return it, not both"), but I'm wondering if it's a good idea to also log it here once it's reached the edge of the system? If not log, maybe send to Sentry, or both? If we get an error at this point I think there's probably a bug somewhere (hence "internal server error"), so it would probably be useful to see that on a Sentry dashboard and get notified in Slack through Sentry.

(Similar comments can probably be made in other endpoints that return 500 errors)